environment.py

```diff
@@ -146,7 +146,7 @@ def env() -> Mapping[str, Optional[str]]:
         # current catalog against, or "*" for all sources in the catalog. Only
         # matching sources will be operated on. If no sources match, an error
         # will be raised.
-        'azul_current_sources': '*',
+        'azul_current_sources': None,
```

Comment:
The comment above should explain why the actual default isn't defined here and what the typical use case is, i.e., when the default value is typically overridden (interactively, on GitLab, when running the job). The comment should  explain where the variable is generally used, similar to the comment on `azul_current_catalog`. And please add "By default, these task are performed on all catalogs." to that earlier comment.